R/module_transform_data.R

```diff
@@ -30,7 +30,7 @@ ui_teal_transform_data <- function(id, transformators, class = "well") {
   lapply(
     names(transformators),
     function(name) {
-      child_id <- NS(id)(name)
+      child_id <- NS(id, name)
```

Comment:
Is there a difference in between those two?

```r
> NS('x', 'a')
[1] "x-a"
> NS('x')('a')
[1] "x-a"
```

Reply:
Readability, the previous code looks awkward